pkg/engine/jmespath/functions_test.go

```diff
@@ -1549,3 +1549,63 @@ UFOZZVoELaasWS559wy8og39Eq21dDMynb8Bndn/
 		})
 	}
 }
+
+func Test_TimeAdd(t *testing.T) {
+	testCases := []struct {
+		test           string
+		expectedResult string
+	}{
+		{
+			test:           "time_add('', '2021-01-02T15:04:05-07:00', '3h')",
+			expectedResult: "2021-01-02T18:04:05-07:00",
+		},
+		{
+			test:           "time_add('Mon Jan 02 15:04:05 MST 2006', 'Sat Jan 02 15:04:05 MST 2021', '5h30m40s')",
+			expectedResult: "2021-01-02T20:34:45Z",
+		},
+	}
+	for i, tc := range testCases {
+		t.Run(fmt.Sprintf("case %d", i), func(t *testing.T) {
+			query, err := New(tc.test)
+			assert.NilError(t, err)
+
+			res, err := query.Search("")
+			assert.NilError(t, err)
+
+			result, ok := res.(string)
+			assert.Assert(t, ok)
+
+			assert.Equal(t, result, tc.expectedResult)
+		})
+	}
+}
+
+func Test_TimeConvert(t *testing.T) {
+	testCases := []struct {
+		test           string
+		expectedResult string
+	}{
+		{
+			test:           "time_convert('', '2021-01-02T15:04:05-07:00')",
```

Comment:
Does it make sense to call `time_convert` with an empty format ?

Reply:
empty string everywhere else means we are using RFC3339, I kept it for consistency

and if the layout is empty and the time isn't in RFC3339 format then it will throw an error

Reply:
I understand the idea but why one would call `time_convert` if the input is already in RFC3339 ?

Reply:
I would rather not default the format, sounds error prone.

Reply:
ok, I should remove it then